package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "todolist_project",
+  "version": "1.0.0",
+  "description": "",
+  "private": true,
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack"
```

Comment:
- [ ] You need to add more scripts, remember that the basics are for development and production, the build script could be for development but you are missing the one for production